### ISSUE TITLE
fix(updownload): Cannot download ontology name and description

### ISSUE THREAD

**Issue by Fernananas** [molgenis/molgenis-emx2#2805]
**What version of EMX2 are you using (see footer)**
v8.218.3

**Describe the bug**
In schema editor you can add label and description to the specific ontology tables. When downloading whole dataset, this information is not downloaded.

**To Reproduce**
1. Go to 'https://cranio-demo.molgeniscloud.org/ERN%20CRANIO/tables/#/'
2. See all labels and descriptions.
3. Go to Up/Download and Export all data as csv
4. Upload zip in empty database
5. See all descriptions and labels are missing

**Expected behavior**
I expect that when downloading whole dataset, that I can upload it again and that is would have the same information. Including label and description of the ontology tables.

**Screenshots**
source:
<img width="1049" alt="Scherm­afbeelding 2023-10-02 om 12 04 41" src="https://github.com/molgenis/molgenis-emx2/assets/40240913/cbe6514e-97d4-417c-b2b3-9db002c82e98">
after upload in empty database:
<img width="1049" alt="Scherm­afbeelding 2023-10-02 om 12 05 45" src="https://github.com/molgenis/molgenis-emx2/assets/40240913/e34243a7-9837-44a4-a45b-6889875c990f">

**Comment by mswertz** [comment]
Right, currently ontology tables are totally filtered from export. We would like to change this to download only the tables, not their columns.

N.B. a future feature request might be that you want to be able to add custom columns, next to the standard ones, and download those as well.
